src/momento/errors/exceptions.py

```diff
@@ -0,0 +1,271 @@
+    ):
+        super().__init__(
+            message,
+            MomentoErrorCode.BAD_REQUEST_ERROR,
+            transport_details,
+            message_wrapper="The request was invalid; please contact us at support@momentohq.com",
+        )
+
+
+class CancelledException(SdkException):
+    """Operation was cancelled."""
+
+    def __init__(
+        self,
+        message: str,
+        transport_details: Optional[MomentoErrorTransportDetails] = None,
+    ):
+        super().__init__(
+            message,
+            MomentoErrorCode.CANCELLED_ERROR,
+            transport_details,
+            message_wrapper="The request was cancelled by the server; please contact us at support@momentohq.com",
+        )
+
+
+class FailedPreconditionException(SdkException):
+    """The server did not meet the precondition to run a command.
+
+    For example, calling `Increment` on a key that doesn't store
+    a number.
+    """
+
+    def __init__(
+        self,
+        message: str,
+        transport_details: Optional[MomentoErrorTransportDetails] = None,
+    ):
+        super().__init__(
+            message,
+            MomentoErrorCode.FAILED_PRECONDITION_ERROR,
+            transport_details,
+            message_wrapper="System is not in a state required for the operation's execution",
+        )
+
+
+class InternalServerException(SdkException):
+    """Momento Service encountered an unexpected exception while trying to fulfill the request."""
+
+    def __init__(
+        self,
+        message: str,
+        transport_details: Optional[MomentoErrorTransportDetails] = None,
+    ):
+        super().__init__(
+            message,
+            MomentoErrorCode.INTERNAL_SERVER_ERROR,
+            transport_details,
+            message_wrapper="An unexpected error occurred while trying to fulfill the request; please contact us at support@momentohq.com",  # noqa: E501
+        )
+
+
+class InvalidArgumentException(SdkException):
+    """SDK client-side validation failed."""
+
+    def __init__(
+        self,
+        message: str,
+        transport_details: Optional[MomentoErrorTransportDetails] = None,
+    ):
+        super().__init__(
+            message,
+            MomentoErrorCode.INVALID_ARGUMENT_ERROR,
+            transport_details,
+            message_wrapper="Invalid argument passed to Momento client",
+        )
+
+
+class LimitExceededException(SdkException):
+    """Requested operation couldn't be completed because system limits were hit."""
+
+    def __init__(
+        self,
+        message: str,
+        transport_details: Optional[MomentoErrorTransportDetails] = None,
+    ):
+        super().__init__(
+            message,
+            MomentoErrorCode.LIMIT_EXCEEDED_ERROR,
+            transport_details,
+            message_wrapper="Request rate exceeded the limits for this account. To resolve this error, reduce your request rate, or contact us at support@momentohq.com to request a limit increase",  # noqa: E501
```

Comment:
The [standards and practices error doc](https://github.com/momentohq/standards-and-practices/blob/main/docs/client-specifications/error-handling.md) has the latest wording for this. It should be "Request rate, bandwidth, or object size exceeded the limits for this account.  To resolve this error, reduce your usage as appropriate or contact us at support@momentohq.com to request a limit increase".

Reply:
Fixed in https://github.com/momentohq/client-sdk-python/pull/179/commits/d76386a245b224eff158c22ca7f9c442e4c6df33

Reply:
Added issue https://github.com/momentohq/client-sdk-dotnet/issues/358